server/models/interfaces.ts

```diff
@@ -369,6 +370,37 @@ export interface GetSnapshotsResponse {
   totalSnapshots: number;
 }
 
+export interface GetIndexRecoveryResponse {
+  response: {
+    shards: [
+      {
+        source: {
+          index: string;
+          repository: string;
+          snapshot: string;
+        };
+        stage: string;
+        start_time_in_millis: number;
```

Comment:
Super nit: _start_time_ms_ instead of _start_time_in_millis_

Reply:
These are the actual keys one finds in the response, so I stuck with it.  Does not appear in UI

Reply:
aah, okay